### DISCUSSION TITLE
Contributing to the v5 documentation

### DISCUSSION THREAD

**Discussion by adam-s** [statelyai/xstate#4428] [General]
Xstate is a fantastic idea. I believe that it will become a standard tool in the Javascript / Typescript ecosystem for state management. Can I help with the v5 documentation? Currently, for v5, I’m stepping through the tests which is tedious. It makes a lot of sense help write documentation while submerged in the tests as I grok the nuance.  Are there suggestions and wisdom to accomplish this, to get changes committed? Are there other people working on the documentation who I can coordinate with?

**Comment by davidkpiano** [comment]
We would love for you to help contribute! I am slowly working on the docs (more so ahead of the release), and you can also coordinate with @audionerd who is helping out as well.

I would also go through the release notes for the `@beta` versions and see what's missing/changed in the docs: https://github.com/statelyai/xstate/releases

**Comment by audionerd** [comment]
Would be great to have more eyes on the documentation! You can track what I’ve been working on so far in the PRs I’ve submitted for this repo and for the docs repo. I’ve been posting progress updates on the Stately Discord documentation channel.
